src/components/Disputes/actions/DisputeVoting.js

```diff
@@ -102,8 +114,56 @@ function VotingActions({ canJurorVote, onRequestCommit }) {
   )
 }
 
-const VotingButton = styled(Button)`
-  width: calc((100% - ${2 * GU}px) / 3);
+const RefuseToVoteHint = ({ compactMode, width }) => {
+  const theme = useTheme()
+
+  const Container = compactMode ? Label : 'div'
+
+  return (
+    <div
+      css={`
+        display: flex;
+        align-items: center;
+        justify-content: ${compactMode ? 'center' : 'flex-end'};
+        margin-bottom: ${1.5 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          width: ${width};
+          color: ${theme.help};
+        `}
+      >
+        <Container
+          style={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+          }}
+        >
+          <span
+            css={`
+              margin-right: ${0.5 * GU}px;
+            `}
+          >
+            Why refuse to vote?
+          </span>
+          <Help hint="">
+            You can refuse to vote for many reasons, for example if you consider
+            that the evidence was not conclusive enough or the description was
+            incoherent. In any case, you won’t be penalized at this stage for
+            selecting any of these options. Remember that you should vote the
+            way that you think a majority of jurors will vote, since you will be
+            penalized if your vote is in the minority.
```

Comment:
> In any case, you won’t be penalized at this stage for selecting any of these options.

Just sanity checking—do we really want to say this? Sure, they won't get penalized _now_, but it's not like this is not an action without repercussion.

-------

Maybe it would be nice to move this into its own paragraph:

> Remember that you should vote the way that you think a majority of jurors will vote, since you will be penalized if your vote is in the minority.

Reply:
> Just sanity checking—do we really want to say this? Sure, they won't get penalized now, but it's not like this is not an action without repercussion.

Yeah i think we could remove this part. For users that are new to the court could easily get the wrong even though we state below that they could be penalized.

> Maybe it would be nice to move this into its own paragraph:

👍